w3/domain/usecase/user_it_usecase.go

```diff
@@ -105,7 +109,7 @@ func (u *userITUsecase) LoginUserIT(request UserITLoginRequest) (UserITLoginResp
 	if err != nil {
 		return UserITLoginResponse{}, err
 	}
-	if !u.bcryptHelper.Compare(user.Password, request.Password) {
+	if !u.bcryptHelper.Compare(user.Password.String, request.Password) {
```

Comment:
ini berubah karena sql.nullstring

Reply:
That's the way to access the real value in sql.NullString type.